client/src/ecash/mod.rs

```diff
@@ -1,82 +1,82 @@
 use super::*;
 
-use escrow_client::EscrowUser;
 use cdk::secp256k1::rand::Rng;
 use cdk::{
-    amount::SplitTarget,
-    cdk_database::WalletMemoryDatabase,
-    nuts::{Conditions, CurrencyUnit, PublicKey, SecretKey, SigFlag, SpendingConditions, Token},
-    wallet::Wallet,
+	amount::SplitTarget,
```

Comment:
The indentation change from 4 to 8, isn't it too much now? Is this the standard in Rust?

Reply:
Seems to be rustfmt standard. I did not change rustfmt settings to format this file except to use tabs instead of spaces (hard_tabs = true in .rustfmt.toml) with the following reasons (grabbed from stackoverflow):

* allow mistakes to be more noticeable, it's annoying to micro manage to spaces.
* are consistent, they are only used for indentation, unless you are commenting wrong.
* are customizable, you can specify in the options of the editor how wide they are.
* are more productive, why type 4 spaces when ye can hit 1 tab.
* take less size than multiple spaces.
* take more space than a space.
* take less time to go through.
* tend to work better in IDEs.